File: project/Resources/Scripts/TimeScale.lua
local timer = 0.0
speed = 1.0
vol = 0.1

function Init()

end

function Update()
    timer = timer + GetDeltaTime()
    transform.scale.x = 1.0 + math.sin(timer*speed) * vol
    transform.scale.y = 1.0 + math.sin(timer*speed) * vol
end

File: project/Resources/XEVIOUS3D/Scripts/TimeScale.cs
using System;
using QuickForgeEngine;

public class TimeScale : QuickForgeComponent
{
    float speed = 3.0f;
    float amplitude = 0.1f;

    float time = 0.0f;
    public override void Initialize(float deltaTime)
    {
    }
    public override void Update(float deltaTime)
    {
        time += deltaTime;
        
        transform_.scale.x = 1.0f + (float)Math.Sin(time * speed) * amplitude;
        transform_.scale.y = 1.0f + (float)Math.Sin(time * speed) * amplitude;
        transform_.scale.z = 1.0f + (float)Math.Sin(time * speed) * amplitude;
    }
}